src/deploy/functions/runtimes/node/parseRuntimeAndValidateSDK.ts

```diff
@@ -24,16 +24,16 @@ const ENGINE_RUNTIMES_NAMES = Object.values(ENGINE_RUNTIMES);
 export const RUNTIME_NOT_SET =
   "`runtime` field is required but was not found in firebase.json.\n" +
   "To fix this, add the following lines to the `functions` section of your firebase.json:\n" +
-  '"runtime": "nodejs16"\n';
+  '"runtime": "nodejs18"\n';
 
 export const UNSUPPORTED_NODE_VERSION_FIREBASE_JSON_MSG = clc.bold(
   `functions.runtime value is unsupported. ` +
-    `Valid choices are: ${clc.bold("nodejs{10|12|14|16}")}.`
+    `Valid choices are: ${clc.bold("nodejs{10|12|14|16|18}")}.`
```

Comment:
TIL that GCF still supports node 10: https://cloud.google.com/functions/docs/concepts/nodejs-runtime